src/test/consensus/LedgerTrie_test.cpp

```diff
@@ -289,6 +289,20 @@ class LedgerTrie_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testEmpty()
+    {
+        using namespace csf;
+        LedgerTrie<Ledger> t;
+        LedgerHistoryHelper h;
+        BEAST_EXPECT(t.empty());
+
+        t.insert(h["abc"]);
+        BEAST_EXPECT(!t.empty());
+        t.remove(h["abc"]);
+        BEAST_EXPECT(t.empty());
```

Comment:
Since the intention of this PR is to distinguish between "none" and "genesis", is there any equivalent to a "genesis" that you could test here and verify that distinction works, too?

Reply:
TL;DR - I will add.

https://github.com/ripple/rippled/pull/2784/files#diff-365742a1743cbbee02d028c406fd931aR830 is a test for "none" vs "genesis", but that is one level removed from `LedgerTrie`.  

https://github.com/ripple/rippled/pull/2784/files#diff-5c7d3493f64f0cbf918b63ba2a47a4c7R362 also halfway covers it.

But I'll add something which explicitly covers the distinction.